packages/web/src/app/AppProviders.tsx

```diff
@@ -67,19 +67,19 @@ export const AppProviders = ({ children }: AppProvidersProps) => {
                   <HeaderContextProvider>
                     <NavProvider>
                       <ScrollProvider>
-                        <ToastContextProvider>
-                          <AppContextProvider>
-                            <AudiusQueryProvider>
-                              <MainContentContextProvider>
-                                <ThemeProvider>
+                        <ThemeProvider>
```

Comment:
@dylanjeffers could you verify this feels ok to you? toast can't use harmony otherwise, but I think this is right

Reply:
Yeah that works great